modules/heat_conduction/src/bcs/GapHeatTransfer.C

```diff
@@ -356,9 +359,9 @@ GapHeatTransfer::computeQpOffDiagJacobian(unsigned jvar)
 
     // THIS IS NOT THE NORMAL WE NEED.
     // WE NEED THE NORMAL FROM THE CONSTRAINT, THE NORMAL FROM THE
-    // MASTER SURFACE.  HOWEVER, THIS IS TRICKY SINCE THE NORMAL
-    // FROM THE MASTER SURFACE WAS COMPUTED FOR A POINT ASSOCIATED
-    // WITH A SLAVE NODE.  NOW WE ARE AT A SLAVE INTEGRATION POINT.
+    // PRIMARY SURFACE.  HOWEVER, THIS IS TRICKY SINCE THE NORMAL
+    // FROM THE PRIMARY SURFACE WAS COMPUTED FOR A POINT ASSOCIATED
+    // WITH A SECONDARY NODE.  NOW WE ARE AT A SECONDARY INTEGRATION POINT.
     //
     // HOW DO WE GET THE NORMAL WE NEED?
```

Comment:
I DON'T KNOW!! AD FOR GEOMETRIC OBJECTS MAYBE?

Reply:
![image](https://user-images.githubusercontent.com/10248304/85640579-0585b680-b641-11ea-8622-52dbfbb67785.png)